src/Widgets/SaveDialog.vala

```diff
@@ -75,20 +76,23 @@ public class Screenshot.SaveDialog : Granite.Dialog {
         });
 
 
-        var preview_box = new Gtk.Grid ();
-        preview_box.halign = Gtk.Align.CENTER;
+        var preview_box = new Gtk.Grid () {
+            halign = Gtk.Align.CENTER
+        };
         preview_box.add (preview_event_box);
 
         unowned Gtk.StyleContext preview_box_context = preview_box.get_style_context ();
         preview_box_context.add_class (Granite.STYLE_CLASS_CARD);
         preview_box_context.add_class (Granite.STYLE_CLASS_CHECKERBOARD);
 
-        var dialog_label = new Gtk.Label (_("Save Image as…"));
+        var dialog_label = new Gtk.Label (_("Save Image as…")) {
+            halign = Gtk.Align.START
+        };
         dialog_label.get_style_context ().add_class (Granite.STYLE_CLASS_H4_LABEL);
-        dialog_label.halign = Gtk.Align.START;
 
-        var name_label = new Gtk.Label (_("Name:"));
-        name_label.halign = Gtk.Align.END;
+        var name_label = new Gtk.Label (_("Name:")) {
```

Comment:
I think it will look better if the `name_label` is properly aligned with the `name_entry`. We may just need to increase the outer grid spacing so that there is room for validation messages on fields that need them?

Reply:
I tried adjusting the layout in the last commit.  I moved the error message revealer above the name entry so that it does not interrupt the entry and comboboxes.